src/DIRAC/WorkloadManagementSystem/Agent/PilotStatusAgent.py

```diff
@@ -119,7 +120,7 @@ def clearWaitingPilots(self, condDict):
 
     for pilotRef in refList:
       self.log.info('Setting Waiting pilot to Stalled: %s' % pilotRef)
-      result = self.pilotDB.setPilotStatus(pilotRef, 'Stalled', statusReason='Exceeded max waiting time')
+      result = self.pilotDB.setPilotStatus(pilotRef, PilotStatus.STALLED, statusReason='Exceeded max waiting time')
```

Comment:
I don't know if I ever seen a pilot in status "Stalled". Is this ever reachable?

Reply:
Indeed, I think the method is not used anymore (`clearWaitingPilots` and also `clearParentJob`).
Should we delete them?
I will at least remove the status `Stalled` from `PilotStatus`.

Reply:
Yes, go ahead and delete